go.mod

```diff
@@ -104,9 +104,9 @@ require (
 	github.com/pkg/errors v0.8.0
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/pquerna/ffjson v0.0.0-20180717144149-af8b230fcd20 // indirect
-	github.com/prometheus/client_golang v0.9.2
-	github.com/prometheus/client_model v0.0.0-20180712105110-5c3871d89910
-	github.com/prometheus/common v0.0.0-20181126121408-4724e9255275
+	github.com/prometheus/client_golang v0.9.4
+	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
```

Comment:
is there a tagged release of `github.com/prometheus/client_model` we should use?

Reply:
nevermind, looks like this is the SHA client_golang uses at v0.9.4 (https://github.com/prometheus/client_golang/blob/v0.9.4/go.mod#L9)